### PR TITLE
AV-1834: ajankohtaista component

### PR DESCRIPTION
**Pull request by teravill**
Updated the ajankohtaista module on the front page
image

### CODE REVIEW

File: drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig [hunk 1]
```diff
@@ -7,89 +7,98 @@
...
                 APIs
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
+
+          <div class="avoindata-explore-card-text">
             <i class="far fa-chevron-right"></i> 
-            <a href="/data/{{ language }}/apiset">
+            <a href="/data/{{ language }}/dataset">
```
Comment (beazil) [inline drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig:63]: Is this link updated from apiset => dataset for a purpose?

File: drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig [hunk 2]
```diff
@@ -7,89 +7,98 @@
...
                 Apps
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
-            <i class="far fa-chevron-right"></i> <a href="/data/{{ language }}/showcase">
+
+          <div class="avoindata-explore-card-text">
+            <i class="far fa-chevron-right"></i> 
+            <a href="/data/{{ language }}/dataset">
```
Comment (beazil) [inline drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig:89]: Is this link updated from showcase => dataset for a purpose?

**Review by beazil — CHANGES_REQUESTED** [review]
Those two links seems to be updated incorrectly point to dataset, otherwise looks good

File: drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig (hunk 1 above)
Comment (teravill) [reply drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig]: Fixed in [a6d7b26](https://github.com/vrk-kpa/opendata/pull/1730/commits/a6d7b268a8685aa016145c4124821dccf0eecd1f)

File: drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig (hunk 2 above)
Comment (teravill) [reply drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig]: Fixed in [a6d7b26](https://github.com/vrk-kpa/opendata/pull/1730/commits/a6d7b268a8685aa016145c4124821dccf0eecd1f)

**Review by beazil — APPROVED** [review]

**Pull request merged**